open-api/swagger.yaml

```diff
@@ -2226,6 +2194,106 @@ paths:
               properties:
                 numberOfUpdatedSets:
                   type: integer
+
+  /v2/tree_nodes/clear_cache:
+    get:
+      description: |
+        Clears tree node and counts caches.
+        Only for administrators.
+        Deprecated. Use /v2/admin/system/after_data_loading_update instead.
+      deprecated: true
+      tags:
+        - v2
+        - admin
+      responses:
+        200:
+          description: |
+            The cache has been cleared.
+
+  /v2/tree_nodes/rebuild_cache:
+    get:
+      description: |
+        Clears tree node and counts caches and rebuilds the tree node cache per user.
+        Only for administrators.
+        Deprecated. Use /v2/admin/system/after_data_loading_update instead.
+      deprecated: true
+      tags:
+        - v2
+        - admin
+      responses:
+        200:
+          description: |
+            Clearing and rebuilding the cache has been started.
+        503:
+          description: |
+            A rebuild operation is already in progress.
+
+  /v2/admin/system/after_data_loading_update:
+    get:
+      description: |
+        This endpoint should be called after loading, deleting or updating data in the database. Clears tree nodes, counts caches, patient sets and bitsets. Updates data for subscribed user queries.
+        Only for administrators.
+      tags:
+        - v2
+        - admin
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/UpdateStatus'
+          description: |
+            The cache and patient sets have been cleared. The data for subscribed user queries is updated.
+
+  /v2/admin/system/update_status:
```

Comment:
Is it a correct description? Looks like a copy of `/v2/admin/system/after_data_loading_update`